optimum/modeling_base.py

```diff
@@ -107,10 +125,10 @@ def save_pretrained(
     @abstractmethod
     def _save_pretrained(self, save_directory, **kwargs):
         """
-        Save a model weights into a directory, so that it can be re-loaded using the
-        [`~onnxruntime.modeling_ort.OptimizedModel.from_pretrained`] class method.
+        Saves a model weights into a directory, so that it can be re-loaded using the
```

Comment:
personally I would keep imperative https://stackoverflow.com/questions/59902102/why-is-imperative-mood-important-for-docstrings

Reply:
Yes, I actually try to fix that as much as I can, thanks for spotting this one!

Reply:
I think it is more about being consistent. We are [supposed to follow the Google format](https://github.com/huggingface/doc-builder#writing-documentation-for-hugging-face-libraries), which specifies descriptive.

Reply:
Oh I did not know that, thanks!